validator/main.py

```diff
@@ -182,35 +188,35 @@ def validate(self, value: Any, metadata: Dict[str, Any]) -> ValidationResult:
             error_spans=error_spans
         )
 
-    # def _inference_local(self, model_input: Any) -> Any:
-    #     """Local inference method running the PII analyzer and anonymizer locally."""
-
-    #     results = self.pii_analyzer.analyze(
-    #         text=model_input["text"], entities=model_input["entities"], language="en"
-    #     )
-    #     results = cast(List[Any], results)
-    #     anonymized_text = self.pii_anonymizer.anonymize(
-    #         text=model_input["text"], analyzer_results=results
-    #     ).text
-    #     return anonymized_text
-
-    # def _inference_remote(self, model_input: Any) -> Any:
-    #     """Remote inference method for a hosted ML endpoint"""
-    #     request_body = {
-    #         "inputs": [
-    #             {
-    #                 "name": "text",
-    #                 "shape": [1],
-    #                 "data": [model_input["text"]],
-    #                 "datatype": "BYTES"
-    #             },
-    #             {
-    #                 "name": "entities",
-    #                 "shape": [len(model_input["entities"])],
-    #                 "data": model_input["entities"],
-    #                 "datatype": "BYTES"
-    #             }
-    #         ]
-    #     }
-    #     response = self._hub_inference_request(json.dumps(request_body), self.validation_endpoint)
-    #     return response["outputs"][0]["data"][0]
+    def _inference_local(self, model_input: Any) -> Any:
+        """Local inference method running the PII analyzer and anonymizer locally."""
+
+        results = self.pii_analyzer.analyze(
+            text=model_input["text"], entities=model_input["entities"], language="en"
+        )
+        results = cast(List[Any], results)
+        anonymized_text = self.pii_anonymizer.anonymize(
+            text=model_input["text"], analyzer_results=results
+        ).text
+        return anonymized_text
+
+    def _inference_remote(self, model_input: Any) -> Any:
+        """Remote inference method for a hosted ML endpoint"""
+        request_body = {
+            "inputs": [
+                {
+                    "name": "text",
+                    "shape": [1],
+                    "data": [model_input["text"]],
+                    "datatype": "BYTES"
+                },
+                {
+                    "name": "pii_entities",
+                    "shape": [len(model_input["entities"])],
+                    "data": model_input["entities"],
+                    "datatype": "BYTES"
+                }
+            ]
+        }
+        response = self._hub_inference_request(json.dumps(request_body), self.validation_endpoint)
```

Comment:
Guys, there is a problem with the response here.
The response now is `{'message': 'Endpoint not found for validator: pii-validator'}`.
Were there any changes made on the server side?